### ISSUE TITLE
Lagging 100

### ISSUE THREAD

**Issue by CCheek** [Mithion/Mana-And-Artifice#22]
I saw that you are adding something to do with rituals and regents, so this might get fixed, but:
If you place a ritual that requires regents, and then destroy the chalk (this doesn't cause an explosion because it's still in the placing regents phase) you can place the ritual again, doubling the amount of ghost items showing the regents, then if you follow through, placing all of the regents, when you activate it, instead of consuming the dust and starting the next part of the ritual, it adds another set of items, for rituals with a lot of regents, if you just hold right click, the world gets really laggy really fast, and there's no way to stop it.

**Comment by Mithion** [comment]
You can shift+right click the center ritual block with purified vinteum dust to abort the existing ritual as a workaround for the time being.

**Comment by Mithion** [comment]
I am not able to reproduce this issue.

I did the following steps:

1.) created a ritual, got it to the placing reagents phase.
2.) instead of placing reagents, broke all chalk runes and re-placed them
3.) re-clicked the middle block with purified vinteum dust

Instead of creating a new ritual, the blocks were updated to purple and the existing ritual continued.

I also tried holding down the right mouse button with purified vinteum, was not able to get a second ritual to spawn.

If you have further steps that I'm missing, or a screenshot/gif/video showing the issue let me know.  If I don't hear anything I'll close in a few days.
